### PR TITLE
update integration and validation readme's

### PR DESCRIPTION
**Pull request by cbron**
I created integration/readme.md because readme.txt is parsed during test setup process.

### CODE REVIEW

**Comment by cbron** [comment]
@bmdepesa I updated validation readme, let me know if thats appropriate layout/details.

**Comment by bmdepesa** [comment]
LGTM (for the validation readme)

File: tests/integration/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+
+#### Notes
+
+To debug, use the standard inline process: `breakpoint()`
+
+The tests use a Rancher python client (https://github.com/rancher/client-python) which is dynamically generated based on the Rancher API, so any methods called on it do not exist until runtime.
+It will be helpful to use the debugger and tools like `dir(admin_mc)` to see all methods on a variable.
+
+conftest.py holds the primary supporting code, see `func admin_mc` for example.
```
Comment (dramich) [inline tests/integration/README.md]: func -> def
This might as well just point to pytest docs vs trying to explain it in one sentence.

Comment (cbron) [reply tests/integration/README.md]: fixed

**Comment by dramich** [comment]
lgtm

**Pull request merged**